aspnetcore/tutorials/grpc/grpc-start.md

```diff
@@ -306,6 +306,9 @@ info: Microsoft.AspNetCore.Hosting.Diagnostics[2]
       Request finished in 78.32260000000001ms 200 application/grpc
 ```
 
+> [!NOTE]
+> The template uses the ASP.NET Core HTTPS development certificate to secure the gRPC service. If the client fails with the message `The remote certificate is invalid according to the validation procedure.` then the development certificate is not trusted on your machine. For instructions to fix this issue, see [Trust the ASP.NET Core HTTPS development certificate on Windows and macOS](xref:security/enforcing-ssl#trust-the-aspnet-core-https-development-certificate-on-windows-and-macos).
```

Comment:
The templates don't use the dev cert. See [Trust the ASP.NET Core HTTPS development certificate on Windows and macOS](https://docs.microsoft.com/en-us/aspnet/core/security/enforcing-ssl?view=aspnetcore-2.2&tabs=visual-studio#trust-the-aspnet-core-https-development-certificate-on-windows-and-macos)

.NET Core SDK includes a HTTPS development certificate that is installed but not tested. To trust the cert, run `dotnet dev-certs https --trust`

Reply:
What I want to get across is the user experience after creating a service using the template is the app will have a HTTPS endpoint that uses the dev cert.

What is a better way of putting that info in the doc?